lib/components/Images/image.css

```diff
@@ -0,0 +1,33 @@
+.image-component {
+    display: flex;
+    flex-flow: column;
+    gap: 1rem;
+}
+
+.image-component iframe{
+    height: 8rem;
+}
+
+.container-img {
+    padding: 1rem;
+    box-shadow: var(--shadow-color) 0px 3px 8px;
+    border-radius: 1rem;
+}
+
+.responsive-img {
+    width: 100%;
+    height: 30rem;
+}
+
+.round-image-container {
+    display: flex;
+    flex-flow: column wrap;
+    justify-content: center;
+    align-items: center;
+}
+
+.round-img {
+    border-radius: 50%;
+    width: 25rem;
+    height: 25rem;
```

Comment:
same here too. also add the aspect ratio property in the round image.

Reply:
Made the required changes